infra/Dockerfile.node

```diff
@@ -2,6 +2,7 @@ FROM node:16-alpine
 
 USER root
 RUN apk update && apk add curl
+RUN apk add --no-cache python3 make g++
```

Comment:
Curious about this new dependency?

Reply:
Adding these dependencies solved a few of the warnings in the logs during the container bill process, I believe they were associated with one of the python libraries that was installed along with the Alpine image.